test/Tests/src/Data/Numbers_Spec.enso

```diff
@@ -224,6 +224,13 @@ spec =
             Integer.parse "-101021010" 2 . should_fail_with Number_Parse_Error
             Integer.parse "123" 128 . should_fail_with Number_Parse_Error
 
+        Test.specify "should be able to invoke methods on Integer via static method call" <|
+            Integer.+ 1 2 . should_equal 3
+            Integer.+ 1 2.5 . should_equal 3.5
+            Test.expect_panic_with (Integer.+ 1.5 1) Type_Error
```

Comment:
This results in `Type_Error` because self is 1.5 which is Decimal, and not Integer